MaterialDesignColors.Wpf/SwatchesProvider.cs

```diff
@@ -36,11 +46,10 @@ public SwatchesProvider()
                     Read(assemblyName, x.SingleOrDefault(y => y.match.Groups["type"].Value == "primary")?.key),
                     Read(assemblyName, x.SingleOrDefault(y => y.match.Groups["type"].Value == "accent")?.key)
                 ))
-                .ToList();
-
+                .ToList());
         }
 
-        public IEnumerable<Swatch> Swatches { get; }
+        public List<Swatch> Swatches { get; }
```

Comment:
This should return IEnumerable<Swatch>; "Depend on abstractions, not concretions"